tests/Resources/ti.ui.view.test.js

```diff
@@ -0,0 +1,804 @@
+		var view;
+		win = Ti.UI.createWindow({ backgroundColor: 'blue' });
+		view = Ti.UI.createView({ width:Ti.UI.FILL, height:Ti.UI.FILL });
+		win.add(view);
+		win.addEventListener('focus', function () {
+			if (didFocus) {
+				return;
+			}
+			didFocus = true;
+
+			try {
+				should(view.backgroundFocusedColor).be.a.String; // undefined on iOS and Android
+				should(view.backgroundFocusedImage).be.a.String;
+				view.backgroundFocusedColor = 'white';
+				view.backgroundFocusedImage = 'Logo.png';
+				should(view.backgroundFocusedColor).be.eql('white');
+				should(view.backgroundFocusedImage).be.eql('Logo.png');
+
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+		win.open();
+	});
+
+	// FIXME Get working on iOS
+	(((utilities.isWindows8_1() && utilities.isWindowsDesktop()) || utilities.isIOS() || utilities.isAndroid()) ? it.skip : it)('backgroundSelectedColor/Image', function (finish) {
+		var view;
+		win = Ti.UI.createWindow({ backgroundColor: 'blue' });
+		view = Ti.UI.createView({ width:Ti.UI.FILL, height:Ti.UI.FILL });
+		win.add(view);
+		win.addEventListener('focus', function () {
+			if (didFocus) {
+				return;
+			}
+			didFocus = true;
+
+			try {
+				should(view.backgroundSelectedColor).be.a.String; // undefined on iOS and Android
+				should(view.backgroundSelectedImage).be.a.String;
+				view.backgroundSelectedColor = 'white';
+				view.backgroundSelectedImage = 'Logo.png';
+				should(view.backgroundSelectedColor).be.eql('white');
+				should(view.backgroundSelectedImage).be.eql('Logo.png');
+
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+		win.open();
+	});
+
+	// FIXME Get working on iOS and Android
+	(((utilities.isWindows8_1() && utilities.isWindowsDesktop()) || utilities.isIOS() || utilities.isAndroid()) ? it.skip : it)('backgroundDisabledColor/Image', function (finish) {
+		var view;
+		win = Ti.UI.createWindow({ backgroundColor: 'blue' });
+		view = Ti.UI.createView({ width:Ti.UI.FILL, height:Ti.UI.FILL });
+		win.add(view);
+		win.addEventListener('focus', function () {
+			if (didFocus) {
+				return;
+			}
+			didFocus = true;
+
+			try {
+				should(view.backgroundDisabledColor).be.a.String; // undefined on iOS and Android
+				should(view.backgroundDisabledImage).be.a.String;
+				view.backgroundDisabledColor = 'white';
+				view.backgroundDisabledImage = 'Logo.png';
+				should(view.backgroundDisabledColor).be.eql('white');
+				should(view.backgroundDisabledImage).be.eql('Logo.png');
+
+				finish();
+			} catch (err) {
+				finish(err);
+			}
+		});
+		win.open();
+	});
+
+	it.androidMissing('backgroundGradient (radial)', function (finish) {
+		var gradient, radialGradient;
+		this.timeout(10000);
+
+		win = Ti.UI.createWindow({
+			backgroundColor: '#fff'
+		});
+
+		gradient = {
+			type: 'radial',
+			startPoint: {
+				x: 50,
+				y: 50
+			},
+			colors: [ 'red', 'blue' ],
+			startRadius: '90%',
```

Comment:
Do we officially support percentage based radius settings? We currently document it as "Number" and not "Number/String".
http://docs.appcelerator.com/platform/latest/#!/api/Gradient-property-startRadius

I didn't implement support for this on Android because I assumed we didn't support this on iOS. It would be nice to have though, but the question is, what's it relative to? The width? The height? The shortest dimension? The longest dimension?

Reply:
We do support it because iOS handles all units in a central class (TiDimension). So we could support it on other platforms as well, but later.

Reply:
Okay, from looking at our iOS code here...
https://github.com/hansemannn/titanium_mobile/blob/8261c1c17d11628c9fe4b0a9452905e378fb254b/iphone/Classes/TiGradient.m#L244

A percentage based radius is relative to the distance from the center of the view to one of its corners (ie: hypotenuse divided by 2). Good to know.

I'm trying to provide an option for devs to draw a circle ring without having to use our "borderRadius" feature. So, we might want to make what the percentage based radius is relative to settable in the future so that the circle won't get clipped.